lib/web_ui/lib/src/engine/bitmap_canvas.dart

```diff
@@ -37,6 +37,8 @@ class BitmapCanvas extends EngineCanvas {
 
   final _CanvasPool _canvasPool;
 
+  html.CanvasRenderingContext2D get context => _canvasPool.context;
```

Comment:
Can you please expose a fillText() api instead. I want to make sure Context2D doesn't leak out.